src/css/_header.scss

```diff
@@ -39,14 +41,38 @@ header {
   .site-title {
     flex: 1;
   }
+
+  #city-dropdown {
+    display: flex;
+    align-items: center;
+    gap: 10px;
```

Comment:
Nit: we've been using `em` rather than `px`. Web dev is not my specialty, but my understanding is that `em` works better for responsive design and things like retina displays. https://www.reddit.com/r/webdev/comments/wlnt6e/should_i_use_emrem_instead_of_px/

Reply:
sgtm, thanks!

Reply:
Honestly I'm finding it difficult to redo my styling using em without breaking the layout.  I've changed for the gap and media query since that seemed to work fine.  I'll confess my CSS workflow (as well as many of the webdevs I know) is basically hack on it until it does what I want but I won't tell you I understand it particular well.  This url that was in the thread you sent seems pretty good for understanding it better: https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/. 

It looks like in most cases you'd switch back and forth anyway.  That all being said I'd say most of the frontend codebases I seen use a mix of the two and px is still used in many places I don't think many web-devs would agree with "never uses px" but maybe I'm wrong.

If it's obvious to you how I can redo this layout using em I'm happy to make the changes (it doesn't seem to be as simple as calculating em and subbing those values in)

Reply:
That's fine with me. Thanks, James!